db/migrate/20221005133804_add_race_gender_columns_to_client.rb

```diff
@@ -0,0 +1,27 @@
+class AddRaceGenderColumnsToClient < ActiveRecord::Migration[6.0]
+  def change
+    add_column :project_clients, :am_ind_ak_native, :boolean, default: false
+    add_column :project_clients, :asian, :boolean, default: false
+    add_column :project_clients, :black_af_american, :boolean, default: false
+    add_column :project_clients, :native_hi_pacific, :boolean, default: false
+    add_column :project_clients, :white, :boolean, default: false
+
+    add_column :clients, :am_ind_ak_native, :boolean, default: false
+    add_column :clients, :asian, :boolean, default: false
+    add_column :clients, :black_af_american, :boolean, default: false
+    add_column :clients, :native_hi_pacific, :boolean, default: false
+    add_column :clients, :white, :boolean, default: false
+
+    add_column :project_clients, :female, :boolean, default: false
+    add_column :project_clients, :male, :boolean, default: false
+    add_column :project_clients, :no_single_gender, :boolean, default: false
+    add_column :project_clients, :transgender, :boolean, default: false
+    add_column :project_clients, :questioning, :boolean, default: false
+
+    add_column :clients, :female, :boolean, default: false
+    add_column :clients, :male, :boolean, default: false
+    add_column :clients, :no_single_gender, :boolean, default: false
+    add_column :clients, :transgender, :boolean, default: false
+    add_column :clients, :questioning, :boolean, default: false
+  end
```

Comment:
Should I remove the old race and gender columns and drop the Race and Gender tables here as well? Should I add as a second migration? Not sure what we typically do when removing tables/columns. @eanders

Reply:
yes, let's drop those columns and tables.  We'll also need to update the gender rules.